bin/test_basics

```diff
@@ -4,7 +4,7 @@ source bin/stack_functions
 
 preamble_setup
 
-bin/package_forch testing
+#bin/package_forch testing
```

Comment:
bug reference

Reply:
Addressed.

Reply:
you should probably put the right bug number in there.

Reply:
Changed.